source/pakker.rst

```diff
@@ -3,3 +3,214 @@
 Gode Pythonpakker
 =================
 
+I dette kapitel beskrive en række gode og anbefalelsesværdige Pythonpakker, der kan hjælpe med at
+løse gængse problemstillinger en udvikler i SDFE ofte vil støde på. I udvælgelsen af pakkerne
+herunder er der lagt vægt på særligt pythonske implementeringer, der benytter sig af koncepter
+introduceret i nyere udgaver af Python.
```

Comment:
> pythonske implementeringer, der benytter sig af koncepter
introduceret i nyere udgaver af Python.

Jeg tror, at det vil være godt, på et tidspunkt, at få uddybet, hvad pythonsk betyder, samt henvise til eller komme med eksempler på disse nyere koncepter, og så få beskrevet dem her eller henvise til eksempler på disse under Bedste praksis-kapitlet.

Hvad tænker du?

Reply:
Jeg har tænkt på at det kunne være fedt med en side et sted der forklarer essensen PEP8 og The Zen of Python. Så tror jeg det kommer lidt af sig selv.

Reply:
Det lyder godt med sådan en side:)